src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -24,6 +37,14 @@ function LineVis(): JSX.Element {
     return <></>;
   }
 
+  const yDomain = extendDomain(dataDomain, 0.01);
+  disableLog(yDomain[0] * yDomain[1] <= 0);
```

Comment:
As the domain of the log scale is the extended domain, the check on domain boundaries must be done on this very domain. It can indeed happen that `dataDomain` spans strictly positive values but `yDomain` does not as it is extended.
We might need to change something here as it means that the log might be disabled in this case whereas the data spans strictly positive values...

Oh, and `scaleLog` works without problem if the domain spans strictly negative values. So we only need to check that the domain does not span positive AND negative (or null) values.